src/lib.rs

```diff
@@ -133,7 +134,7 @@ impl Alpm {
   /// It behaves like `pacman -Q {query}`
   pub fn query_package_version<S>(&self, query: S) -> std::io::Result<String> where S: Into<String> {
     let s: String = query.into();
-    let mut cs = s.into_bytes();
+    let mut cs = s.as_bytes().to_vec();
```

Comment:
The problem is I don't know how to print the package name if we loose ownership of `s`.